### PR TITLE
Kalypso-Release

### PR DESCRIPTION
**Pull request by tommartensen**

### CODE REVIEW

File: src/SettingsView/EventQueryListItem.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		this.deleteEventQuery = this.deleteEventQuery.bind(this);
+		this.deleteEventQueryButton = this.deleteEventQueryButton.bind(this);
+	}
+	
+	deleteEventQuery() {
+		this.props.deleteQuery(this.props.query);
+	}
+
+	editEventQuery() {
+		// TODO: to be filled
```
Comment (hpibptlab) [inline src/SettingsView/EventQueryListItem.js:23]: INFO Complete the task associated to this TODO comment. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1135)

File: src/Utils/ChangeNotifier.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+			(notification.ArtifactType === "Entity" && subscription.artifactId === notification.ArtifactId)
+			||
+			(notification.ArtifactType === "Event" && subscription.artifactId === parseInt(notification.EntityId, 10))
+		) {
+			subscription.notificationCallback(notification.EventTypeId);
+		}
+	}
+
+	handleNewConnection() {
+		console.log("[WEBSOCKET] Established a new connection! Connection is open.");
```
Comment (hpibptlab) [inline src/Utils/ChangeNotifier.js:41]: MAJOR Remove this logging statement. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)

File: src/Utils/ChangeNotifier.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		const serverNotifications = JSON.parse(event.data);
+		serverNotifications.forEach((notification) => {
+			this.subscribers.forEach((subscription) => {
+				ChangeNotifier.handleNotification(subscription, notification);
+			});
+		});
+	}
+
+	handleConnectionClose() {
+		console.log("[WEBSOCKET] Connection closed!");
```
Comment (hpibptlab) [inline src/Utils/ChangeNotifier.js:58]: MAJOR Remove this logging statement. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)

File: src/Utils/ChangeNotifier.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+			subscription.notificationCallback(notification.EventTypeId);
+		}
+	}
+
+	handleNewConnection() {
+		console.log("[WEBSOCKET] Established a new connection! Connection is open.");
+	}
+	
+	handleError() {
+		console.log("[WEBSOCKET] An error occured! Connection could not be established.");
```
Comment (hpibptlab) [inline src/Utils/ChangeNotifier.js:45]: MAJOR Remove this logging statement. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 3 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by julianweise — APPROVED** [review]

**Pull request merged**